app/assets/stylesheets/static_pages.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the StaticPages controller here.
```

Comment:
You can delete this file if you dont use it

Reply:
not fix yet